config.md

```diff
@@ -46,7 +46,8 @@ The parameters are similar to the ones in [the Linux mount system call](http://m
 * **`destination`** (string, required) Destination of mount point: path inside container.
 * **`type`** (string, required) Linux, *filesystemtype* argument supported by the kernel are listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660"). Windows: ntfs
 * **`source`** (string, required) a device name, but can also be a directory name or a dummy. Windows, the volume name that is the target of the mount point. \\?\Volume\{GUID}\ (on Windows source is called target)
-* **`options`** (list of strings, optional) in the fstab format [https://wiki.archlinux.org/index.php/Fstab](https://wiki.archlinux.org/index.php/Fstab).
+* **`options`** (list of strings, optional) in a platform-appropriate format for each operating system.
+For the Linux OS, if present, options MUST be in the fstab format [https://wiki.archlinux.org/index.php/Fstab](https://wiki.archlinux.org/index.php/Fstab).
```

Comment:
This looks fine to me, although it would be nice to specify what other platforms expect so authors know what to put in here.  @jlb13, @anuthan, can you clarify what Solaris wants here?

Also, I'd like to replace the Arch link with a [link to the man page][1], but I'm happy to file that in a subsequent PR if you don't want to roll it in here.

[1]: http://www.man7.org/linux/man-pages/man5/fstab.5.html